eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -85,6 +86,9 @@ class ZmqWatcher(blockCount: AtomicLong, client: ExtendedBitcoinClient)(implicit
           blockCount.set(count)
           context.system.eventStream.publish(CurrentBlockCount(count))
       }
+      client.rpcClient.invoke("getbalance").collect {
+        case JDecimal(balance) => Metrics.BitcoinBalance.withoutTags().update(balance.doubleValue() * 1000)
```

Comment:
Why do you do `* 1000` ?

Reply:
Because Kamon does a `floor`, so if your balance is `4.99 BTC` it will show you only `4 BTC`.
A balance difference of `0.99 BTC` is a big change that we don't want to miss :)

It's worth a comment though, and maybe I can rename the metric to `MilliBitcoinBalance` or something like that?

Reply:
Maybe

```scala
val BitcoinBalance = Kamon.gauge("bitcoin.balance", "Bitcoin balance (mBTC)")
```

Reply:
Done, it's a pity Kamon doesn't let us define our own measurement units (it only supports standard units like time units, file size units and number of instances of something).